src/libraries/Common/src/Interop/Windows/BCrypt/Interop.BCryptOpenAlgorithmProvider.cs

```diff
@@ -12,7 +11,31 @@ internal static partial class Interop
     internal static partial class BCrypt
     {
         [LibraryImport(Libraries.BCrypt, StringMarshalling = StringMarshalling.Utf16)]
-        internal static partial NTSTATUS BCryptOpenAlgorithmProvider(out SafeBCryptAlgorithmHandle phAlgorithm, string pszAlgId, string? pszImplementation, BCryptOpenAlgorithmProviderFlags dwFlags);
+        internal static partial NTSTATUS BCryptOpenAlgorithmProvider(
```

Comment:
Can this be private now?

Reply:
There's still one call that was slightly complicated, and updating it required adding a try/finally.  So I left it alone.